### PR TITLE
added floor_ frontend function

### LINKED ISSUE #22648: floor_

### PR DESCRIPTION
**Pull request by Aaryan562**
Close #22648

### CODE REVIEW

**Comment by Aaryan562** [comment]
@AlexandraPosta Could you pls review my PR. Thanks!

File: ivy/functional/frontends/paddle/tensor/math.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # local
-from ..math import *  # noqa: F401
+from ..math import exp, floor, lerp, reciprocal, sqrt  # noqa: F401
```
Comment (AlexandraPosta) [inline ivy/functional/frontends/paddle/tensor/math.py:2]: I would not change this line, keep the changes to the functions that you contribute to.

File: ivy/functional/frontends/paddle/tensor/math.py [hunk 2]
```diff
@@ -21,6 +21,12 @@ def exp_(x, name=None):
     return ivy.inplace_update(x, exp(x))
 
 
+@with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
```
Comment (AlexandraPosta) [inline ivy/functional/frontends/paddle/tensor/math.py:24]: Please check the PaddlePaddle documentation for supported types.

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py [hunk 3]
```diff
@@ -3,10 +3,6 @@
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 
 
-# --- Main --- #
-# ------------ #
-
-
 # ceil_
```
Comment (AlexandraPosta) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py:6]: Again, please do not change code that it is not related to your functions
